modules/configuring-high-availability/proc-configuring-high-availability-in-rhdh-operator-deployment.adoc

```diff
@@ -0,0 +1,25 @@
+[id="proc-configuring-high-availability-in-rhdh-operator-deployment"]
+= Configuring High Availability in a {product} Operator deployment
+
+When you are deploying {product-short} using the Operator, you must set `replicas` to a value greater than `1` in your `{product-custom-resource-type}` CR.
+
+
+.Procedure
+
+. In your `{product-custom-resource-type}` CR, set `replicas` to a value greater than `1`. For example:
++
+====
+[source,yaml,subs="+attributes,+quotes"]
+----
+apiVersion: rhdh.redhat.com/v1alpha3 
+               kind: Backstage 
+               metadata: 
+                  name: rhdh-spec: 
+               application: 
+                 ... 
+                replicas:3 <1>
+                 …
+
```

Comment:
Review code formatting (indentation, ellipsis when trucating). See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#yaml-formatting-for-kubernetes-and-openshift-api-objects